docs/Recursion/generate-parantheses.md

```diff
@@ -0,0 +1,65 @@
+---
+id: Generate-Parentheses-problem-dsa
+title: Generate Parentheses Recursion
+sidebar_label: Generate Parentheses
+description: "The Generate Parentheses problem requires generating all combinations of well-formed parentheses given n pairs. The solution uses recursion and backtracking to ensure that each combination is valid."
+tags: [generate-parentheses, backtracking, recursion, dsa]
+---
+
+## Generate Parentheses Problem | Return All Combinations of Well-Formed Parentheses
+
+- Problem Statement: Given n pairs of parentheses, the task is to generate all possible combinations of well-formed parentheses. A well-formed parentheses string must have an equal number of opening and closing brackets, with every opening bracket correctly paired with a closing one.
+
+- The solution involves recursively constructing valid strings by adding opening and closing brackets while maintaining balance.
+
+- Example: For n = 3, the output should be: ["((()))", "(()())", "(())()", "()(())", "()()()"].
```

Comment:
replace `For n = 3, the output should be: ["((()))", "(()())", "(())()", "()(())", "()()()"]` with ``` `For n = 3, the output should be: ["((()))", "(()())", "(())()", "()(())", "()()()"]` ```